devops  alona

```diff
@@ -0,0 +1,20 @@
+from flask import Flask, request
+import subprocess
+
+app = Flask(__name__)
+
+@app.route('/system-info', methods=['GET'])
```

Comment:
You'll see, but we are using the flask_restful library to make things more organized and readable, in addition to its own benefits. lookup the 'res' file as well